config/locales/es.yml

```diff
@@ -1802,6 +1802,23 @@ es:
+              </ul>
+              <p>If I have filed a balance due return, I understand that if the ADOR does not receive full and timely payment of my tax liability by April 18, 2023, I will remain liable for the tax liability and all applicable interest and penalties. When electronically filing my federal and state tax returns, I understand that if there is an error on my federal return, my state return will also be rejected.</p>
+              <p>I consent to my Electronic Return Originator (ERO) or On-Line Service Provider (OLSP) sending my electronic Arizona individual income tax return and accompanying schedules and statements to ADOR, and I consent to my ERO or OLSP sending such information to ADOR through a transmitter. I consent to ADOR sending my ERO, OLSP and/or transmitter an acknowledgement of receipt of transmission and an indication of whether or not the transmission of my return is accepted and, if the return is rejected, the reason(s) for the rejection. If the processing of my return or refund is delayed, I authorize ADOR to disclose to my ERO, OLSP and/ or transmitter the reason(s) for the delay, or when the refund was sent. If ADOR contacts my ERO for a copy of my return, any documents or schedules to my return, and/or this authorization form, I authorize my ERO to release copies of the requested documents to ADOR.</p>
+              <p>I authorize (ELECTRONIC RETURN ORIGINATOR) to make the election that I want my electronic signature to my electronic federal individual income tax return to serve as my signature to my electronic Arizona individual income tax return for the year ending December 31, 2022. I understand that when my ERO makes the election that my electronic signature to my federal individual income tax return will serve as my signature to my Arizona individual income tax return, I will have signed my Arizona individual income tax return and declared under penalties of perjury that to the best of my knowledge and belief the return is true, correct and complete.</p>
+          ny:
+            body_html: |
+              <p>By checking the box shown below, I declare, under penalty of perjury, that I have examined the information on my 2023 New York State electronic personal income tax return, including any accompanying schedules, attachments, and statements, and certify that my electronic return is true, correct, and complete.</p>
+              <p>If I am paying my New York State personal income taxes owed by electronic funds withdrawal, I authorize the New York State Tax Department and its designated financial agents to initiate an electronic funds withdrawal from the financial institution account indicated on my 2023 electronic return, and I authorize my financial institution to debit the entry to my account. I understand and agree that I may revoke this authorization for payment only by contacting the Tax Department no later than five (5) business days prior to the payment date.</p>
```

Comment:
You added values to es.yml. I've been leaving the es.yml keys out for translations that we don't have yet. should I be adding placeholder values like this?

Reply:
@bengolder I only added it because these are _html fields and they wouldn't pass the missing tags test, I don't think you have to worry about adding es.yml if the tests are passing